### PR TITLE
Updated manual stacking implementation

### PR DESCRIPTION
**Pull request by niemyjski**
https://github.com/exceptionless/Exceptionless.Net/pull/82

### CODE REVIEW

File: Source/Core/Pipeline/010_AssignToStackAction.cs [hunk 1]
```diff
@@ -58,7 +58,8 @@ public class AssignToStackAction : EventPipelineActionBase {
                         Logger.Trace().Message("Creating new event stack.").Write();
                         ctx.IsNew = true;
 
-                        string title = _formattingPluginManager.GetStackTitle(ctx.Event);
+                        var msi = ctx.Event.GetManualStackingInfo();
+                        string title = !String.IsNullOrWhiteSpace(msi?.Title) ? msi.Title : _formattingPluginManager.GetStackTitle(ctx.Event);
```
Comment (niemyjski) [inline Source/Core/Pipeline/010_AssignToStackAction.cs]: Was wondering if I should make this into a stack formatting plugin... But I think this is fine.

**Comment by niemyjski** [comment]
@adamzolotarev thoughts?

**Comment by niemyjski** [comment]
My only thoughts is that the model and known key should be changed from StackingInfo to ManualStackingInfo thoughts?

**Comment by adamzolotarev** [comment]
Looks nice, thank you!

**Pull request merged**